src/mpcCoreKit.ts

```diff
@@ -736,7 +738,7 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
     } = generateTSSEndpoints(torusNodeTSSEndpoints, parties, clientIndex, nodeIndexes);
 
     // Setup sockets.
-    const sockets = await setupSockets(tssWSEndpoints, randomSessionNonce);
+    const sockets = await setupSockets(tssWSEndpoints, randomSessionNonce, "/tss/socket.io", this.socketTransports);
```

Comment:
Is it intentional that you set the path here? Otherwise you can also insert undefined and it will fallback to the default as before.